src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -1,5 +1,27 @@
 package mate.jdbc.util;
 
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
 public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load JDBC driver for MySQL", e);
+        }
+    }
 
+    public static Connection getConnection() {
+        try {
+            Properties dbProperties = new Properties();
+            dbProperties.put("user", "root");
+            dbProperties.put("password", "Aktersobaka1992");
+            return DriverManager.getConnection("jdbc:mysql://localhost:3306/init_db", dbProperties);
```

Comment:
@EliosPride init_db is bad name for database schema, name it like 'taxi' or 'taxi_service' or 'taxi_app'.
You had to create init_db.sql file to run it and init your empty DB with some start data (from your task - Create init_db.sql file in src/main/resources folder.)